addons/im_livechat/static/src/views/livechat_looking_for_help_controller_mixin.js

```diff
@@ -0,0 +1,48 @@
+import { LivechatViewControllerMixin } from "@im_livechat/views/livechat_view_controller_mixin";
+
+import { onWillDestroy } from "@odoo/owl";
+import { deserializeDateTime } from "@web/core/l10n/dates";
+
+import { useService } from "@web/core/utils/hooks";
+import { useDebounced } from "@web/core/utils/timing";
+
+const RELOAD_DEBOUNCE_DELAY = 300;
+
+export const LivechatLookingForHelpReloadMixin = (ViewController) =>
+    class extends LivechatViewControllerMixin(ViewController) {
+        setup() {
+            super.setup(...arguments);
+            const busService = useService("bus_service");
+            const reload = useDebounced(() => this.model.load(), RELOAD_DEBOUNCE_DELAY);
+            const onUpdate = ({ added_channel_ids, removed_channel_ids, dt }) => {
+                if (this.lastLoadDt > deserializeDateTime(dt)) {
+                    return;
+                }
+                if (added_channel_ids.length) {
+                    // Filter/Groups/Search are handled server side, it's easier to reload the data
+                    // rather than trying to guess where/if the record should be inserted.
+                    reload();
+                    return;
+                }
+                const recordIdsToRemove = this.model.root.records
+                    .filter(({ resId }) => removed_channel_ids.includes(resId))
+                    .map(({ id }) => id);
+                if (recordIdsToRemove.length) {
+                    this.model.root._removeRecords(recordIdsToRemove);
+                }
+            };
+            busService.addChannel("im_livechat.looking_for_help");
+            busService.subscribe("im_livechat.looking_for_help/update", onUpdate);
+            onWillDestroy(() => {
+                busService.unsubscribe("im_livechat.looking_for_help/update", onUpdate);
+                busService.deleteChannel("im_livechat.looking_for_help");
+            });
+        }
+
+        get modelParams() {
+            const params = super.modelParams;
+            params.hooks.onRootLoaded = () =>
+                (this.lastLoadDt = luxon.DateTime.now().set({ milliseconds: 0 }));
```

Comment:
This value should normally be provided by the server to "sign" the data it sent at the correct time, this way of going it is prone to many issues (late rpc result vs bus, unsync client time, ...)

Reply:
We might want to just do extra reloads, and only discard when bus dt is older than another bus dt.

Reply:
This is mainly used to prevent reloads when first accessing the page, I think I can achieve the same by ensuring there are actually `added_channel_ids` we don't know of yet. If so, we reload. If not, the view is already up to date.